ui/pages/v2/Sources.tsx

```diff
@@ -17,7 +17,7 @@ function SourcesList({ className }: Props) {
       loading={isLoading}
       className={className}
     >
-      <SourcesTable sources={sources} />
+      <SourcesTable sources={sources || []} />
```

Comment:
This looks like a leaky abstraction. Can we make `<SourcesTable />` handle `undefined` instead?

Reply:
I'm going to remove this and handle it when I do the filtering. It has nothing to do with the Nav I was just checking out the errors